src/modules/Common/components/Datatables/UsernameTable.tsx

```diff
@@ -28,6 +34,24 @@ const UsernameTable = ({ exportName, data, onUsernameClick, nbData }: UsernameTa
       size: 1,
       Cell: ({ value }: any) => value.toLocaleString('en'),
     },
+    {
+      Header: 'Action',
+      align: 'right',
+      Cell: ({ ...rest }: any) => {
+        return (
+          <Link href={`/user/@${rest.row?.original?.label}`}>
+            <button
+              className="fr-btn fr-btn fr-btn--sm fr-btn--secondary fr-fi-search-line"
```

Comment:
button should always be of type button
because if ever put in a form, it will submit the form on click

Reply:
fixed, we should do a feedback to DSE team because the documentation does not mention `type=button`
the documentation does not mention